tools/deploy/export_model.py

```diff
@@ -187,9 +190,29 @@ def eval_wrapper(inputs):
     DetectionCheckpointer(torch_model).resume_or_load(cfg.MODEL.WEIGHTS)
     torch_model.eval()
 
-    # get a sample data
-    data_loader = build_detection_test_loader(cfg, cfg.DATASETS.TEST[0])
-    first_batch = next(iter(data_loader))
+    if args.sample_image is None:
+        # get a sample data
+        data_loader = build_detection_test_loader(cfg, cfg.DATASETS.TEST[0])
+        first_batch = next(iter(data_loader))
+    else:
+        # get a sample data
+        original_image = cv2.imread(args.sample_image)
```

Comment:
please use `detection_utils.read_image` to be consistent with how dataset is loaded. It will also support input_format conversion.